templates/web/base/auth/general.html

```diff
@@ -42,6 +42,7 @@
 
                 errors = {
                     nonmobile = loc('Please enter a mobile number'),
+                    sms_failed = tprintf('Sending a confirmation text failed: "%s"', sms_error),
```

Comment:
And thinking a bit more perhaps extract this and the identical one above into a partial so we're only defining these in one place?